data/levels/halloween2014/pools.stl

```diff
@@ -1807,18 +1243,18 @@
+      0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 3084 3087 0 3076 3088 3086 3087 0 3084 3087 0 0 0 0 0 0 3076 3077 3078 3078 3082 3082 3081 3077 3078 3078 3077 3082 3081 3082 3077 3078 3081 3082 3081 3077 3077 3078 3077 3078 3078 3078 3082 3077 3077 3082 3077 3082 3081 3082 3081 3082 3077 3082 3082 3078 3078 3077 3082 3082 3081 3081 3082 3078 3077 3077 3082 3077 3077 3077 3081 3077 3082 3078 3078 3078 3077 3077 3082 3082 3078 3082 3082 3077 3077 3078 3082 3081 3077 3077 3082 3094 3095 3078 3078 3077 3081 3077 3081 3078 3082 3077 3082 3077 3082 3077 3081 3077 3082 3077 3082 3081 3082 3077 3077 3078 3081 3081 3077 3077 3082 3077 3082 3078 3078 3078 3077 3082 3081 3081 3077 3077 3078 3078 3077 3081 3077 3078 3077 3078 3078 3077 3081 3077 3082 3082 3092 3073 3074 3075 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 44 0 0 0 44 0 0 0 0 44 0 0 0 44 0 0 0 0 0 0 0 0 0 0 0 0 0 3072 3073 3074 3075 0 0 0 0 0 0 0 0 0 3072 3073 3093 3077 3082 3081 3077 3077 3077 3077 3082 3082 3081 3082 3081 3077 3077 3077 3082 3081 3078 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 3077 3078 3077 3078 3082 3078 3077 3082 3077 3078 3077 3094 3095 3082 3078 3077 3077 3077 3078 3077 3082 3081 3077 3078 3077 3096 3097 3082 3082 3081 3081 3078 3094 3095 3081 3104 3105 3082 3078 3081 3081 3077 3078 3082 3077 3078 3096 3097 3077 3082 3077 3081 3082 3082 3078 3081 3078 3082 3081 3078 3077 3077 3092 3074 3073 3074 3073 3074 3073 3074 3073 3074 3073 3074 3073 3074 3073 3074 3073 3074 3073 3074 3073 3074 3078 3081 3094 3095 3078 3077 3077 3082 3081 3081 3082 3082 3082 3081 3081 3082 3081 3078 3082 3082 3082 3078 3078 3078 3077 3078 3077 3078 3077 3077 3078 3078 3078 3081 3078 3081 3081 3081 3081 3078 3077 3077 3081 3082 3081 3081 3077 3077 3077 3078 3077 3078 3078 3077 3077 3077 3078 3081 3078 3077 3081 3077 3077 3082 3077 3082 3081 3077 3077 3077 3082 3082 3078 3081 3077 3078 3082 3082 3081 3081 3081 3077 3081 3081 3082 3078 3078 3082 3081 3077 3078 3077 3081 3081 3077 3078 3082 3081 3081 3078 3081 3082 3082 3082 3077 3082 3077 3077 3082 3077 3081 3081 3082 3078 3077 3078 3081 3077 3078 3082 3081 3081 3081 3077 3081 3082 3077 3077 3081 3082 3077 3082 3077 3077 3081 3081 3092 3073 3075 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 3080 3081 3081 3082 3082 3081 3077 3102 3103 3081 3082 3077 3082 3081 3077 3077 3077 3077 3077 3077 3078 3077 3082 3081 3082 3094 3095 3078 3078 3077 3082 3077 3081 3081 3082 3081 3078 3077 3081 3081 3081 3078 3077 3096 3097 3081 3078 3077 3078 3081 3082 3078 3094 3095 3082 3082 3077 3077 3078 3082 3082 3077 3081 3096 3097 3078 3081 3078 3082 3102 3103 3081 3082 3077 3082 3081 3082 3078 3078 3082 3078 3077 3082 3078 3077 3081 3081 3077 3082 3078 3082 3082 3082 3082 3081 3081 3078 3077 3077 3081 3077 3078 3077 3077 3077 3077 3081 3081 3081 3078 3077 3077 3078 3082 3077 3081 3094 3095 3078 3082 3082 3081 3081 3081 3082 3081 3078 3082
```

Comment:
What are the tile changes?

Reply:
I removed the lava from the untinted "Interactive" tilemap and put them to the new green-tinted tilemap.